Assets/Scripts/IngameDebugConsole/Scripts/Commands/ItemCommands.cs

```diff
@@ -0,0 +1,89 @@
+﻿using SS3D.Core;
+using SS3D.Core.Settings;
+using SS3D.Systems.Entities;
+using SS3D.Systems.Inventory.Items;
+using SS3D.Systems.PlayerControl;
+using SS3D.Systems.Inventory.Containers;
+using UnityEngine;
+using SS3D.Systems;
+
+namespace IngameDebugConsole.Commands
```

Comment:
those command should probably be inheriting from Command, and be in the appropriate folder. The command class is integrated with our own systems, the IngameDebugConsole is an external asset. Sorry about the confusion ...